pandas/core/strings.py

```diff
@@ -1180,19 +1180,36 @@ def str_slice(arr, start=None, stop=None, step=None):
 
 def str_slice_replace(arr, start=None, stop=None, repl=None):
     """
+    Replace a sliced string.
+
     Replace a slice of each string in the Series/Index with another
     string.
 
     Parameters
     ----------
     start : int or None
+        Left edge index.
```

Comment:
Can you indicate what None means? (I suppose this is the from the start of the string)

Can you also change "int or None" into "int, optional"? And then maybe you can say for the above "If not specified, slice from the start of each string"

(and same for end, but then "slice until the end of each string"